test/contracts/boilerplate/base.js

```diff
@@ -69,4 +73,34 @@ describe('Base Contract', () => {
       await expect(head2.methods.context().call()).to.be.eventually.rejected;
     });
   });
+
+  describe('funds migration', async () => {
+    const storedValue = '100';
+    const allocateFunds = (from, to, value) => web3.eth.sendTransaction({from, to, value, gas: 1000000});
+    const fundsMigration = (executor, to) => () => calledContract.methods.migrateFunds(to).send({from: executor});
```

Comment:
`async`?

Reply:
also: we typically use `sender` instead of `executor` 😛 

https://www.youtube.com/watch?v=6P69QJ6-LVs